### ISSUE TITLE
Query about circuit operation

### ISSUE THREAD

**Issue by tirthbha** [ubsuny/23-Homework1G2#64]
@Pranjal-Srivastava-2023 @s4il3sh Do I need to add explanation for the other circuit operations too? I have added explanations for 1+1 and 0+1.

**Comment by Pranjal-Srivastava-2023** [comment]
Yes! but the file has conflicts. I will resolve it and merge.

**Comment by tirthbha** [comment]
> Yes! but the file has conflicts. I will resolve it and merge.

@Pranjal-Srivastava-2023 However, the other figures attached to the documentation file are different from those I have commented.

**Comment by Pranjal-Srivastava-2023** [comment]
@tirthbha I just merged patch2.

**Comment by tirthbha** [comment]
@Pranjal-Srivastava-2023 final update is on the "final updated file" branch.

**Comment by s4il3sh** [comment]
yes @tirthbha the last one 0+0 is left. Please complete this too.

**Comment by tirthbha** [comment]
@s4il3sh @Pranjal-Srivastava-2023 
Added comment for 1+0 and 0+0.
